instabot/bot/bot_like.py

```diff
@@ -48,8 +51,8 @@ def like_medias(self, medias):
     for media in tqdm(medias):
         if not self.like(media):
             self.error_delay()
-            broken_items = medias[medias.index(media):]
-            break
+            broken_items.append(media)
+            # break
```

Comment:
Remove this commented line if not needed.

Reply:
done